components/org.wso2.carbon.identity.idp.metadata.saml2/src/main/java/org/wso2/carbon/identity/idp/metadata/saml2/bean/SAMLMetadataErrorResponse.java

```diff
@@ -1,31 +1,54 @@
+/*
+ *  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Add the correct license header.

Reply:
The correct license was added with https://github.com/wso2-extensions/identity-metadata-saml2/pull/68/commits/c7e857ed1c2d376c8e5d44191eeaee5c42c24eab but the year was kept to 2016 since the file was first committed in that year.